src/wp-includes/block-editor.php

```diff
@@ -635,7 +635,7 @@ function get_block_editor_settings( array $custom_settings, $block_editor_contex
 
 	$post_content_block_attributes = wp_get_post_content_block_attributes();
 
-	if ( ! empty( $post_content_block_attributes ) ) {
+	if ( isset( $post_content_block_attributes ) ) {
```

Comment:
I'm not sure this change is the right one. Now that it's using `isset`, it would also enter the condition if no attributes are present on the post content block, since the default return value is an array.

Reply:
Oooh I see. The intention is for `$post_content_block_attributes` to be an empty array if the Post Content block exists but doesn't have any attributes set (that's the expected state of it when it only has default layout). 

I'm thinking the best solution would be to have `wp_get_post_content_block_attributes` return `null` by default instead of an empty array. The function is public though it's only been in core since 6.3; might that present any back compat problems? I did a quick search in wpdirectory and it didn't turn up anything.

Reply:
Actually the function is marked private 😄 so it shouldn't be a problem. I made the change to the default return value.